packages/expo-doctor/src/checks/AppConfigFieldsNotSyncedToNativeProjectsCheck.ts

```diff
@@ -0,0 +1,80 @@
+import fs from 'fs';
+import path from 'path';
+
+import { DoctorCheck, DoctorCheckParams, DoctorCheckResult } from './checks.types';
+import { learnMore } from '../utils/TerminalLink';
+import { isFileIgnoredAsync } from '../utils/isFileIgnoredAsync';
+
+const appConfigFieldsToSyncWithNative = [
+  'ios',
+  'android',
+  'plugins',
+  'icon',
+  'scheme',
+  'userInterfaceStyle',
+  'splash',
+  'updates',
+  'orientation',
+  'backgroundColor',
+  'primaryColor',
+  'notification',
+  'androidStatusBar',
+  'androidNavigationBar',
+  'jsEngine',
+  'locales',
+];
+
+export class AppConfigFieldsNotSyncedToNativeProjectsCheck implements DoctorCheck {
+  description = 'Check for app config fields that may not be synced in a non-CNG project';
+
+  sdkVersionRange = '*';
+
+  async runAsync({
+    exp,
+    projectRoot,
+    staticConfigPath,
+    dynamicConfigPath,
+  }: DoctorCheckParams): Promise<DoctorCheckResult> {
+    const issues: string[] = [];
+    const appJsonFields: string[] = Object.keys(exp);
+    const unsyncedFields: string[] = [];
+    const prebuildMessage: string = fs.existsSync('eas.json')
+      ? 'EAS Build will not sync the following properties:'
+      : `if you don't use Prebuild in your build pipeline, the following properties will not be synced:`;
```

Comment:
```suggestion
      : `if you don't run prebuild in your build pipeline, the following properties will not be synced:`;
```
splitting the difference in my hemming and hawing about how to propertly describe this... in the context of a custom, "running" would be the most accurate, since you would have to explicitly do so.